transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -232,16 +232,20 @@ static jint _socket(JNIEnv* env, jclass clazz, int type) {
 int netty_unix_socket_initSockaddr(JNIEnv* env, jbyteArray address, jint scopeId, jint jport,
                                    const struct sockaddr_storage* addr, socklen_t* addrSize) {
     uint16_t port = htons((uint16_t) jport);
+    // 24 is the maximum possible
```

Comment:
128 bit (16 byte) for IPv6 address plus long port 64 bit (8 byte)?

Reply:
actually we only need 16 here, so I changed it and added a comment. Good catch!